evap/evaluation/migrations/0085_delete_cronjob_user.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.0.9 on 2018-10-15 19:08
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def create_cronjob_user(apps, _schema_editor):
+    UserProfile = apps.get_model('evaluation', 'UserProfile')
+
+    UserProfile.objects.create(username="cronjob")
+
+
+def delete_cronjob_user(apps, _schema_editor):
+    UserProfile = apps.get_model('evaluation', 'UserProfile')
+
+    UserProfile.objects.get(username="cronjob").delete()
```

Comment:
this migration fails because the `UserProfileManager` excludes the cronjob user by default (see https://github.com/fsr-itse/EvaP/blob/master/evap/evaluation/models.py#L882).
please make sure to remove all occurrences in the `UserProfileManager` as well.

Reply:
@janno42 I think I took care of that: https://github.com/fsr-itse/EvaP/pull/1269/files#diff-293b09da50fa9fc3f6f832c04020b4c9L881

Reply:
can you run the migration locally? it fails on my testing system.

Reply:
I deleted my local database, recreated it and ran all migrations successfully. Does a user with username `cronjob` exist on your testing system before this migration is applied?

Reply:
Interesting. I had this failure too, though it was because I messed up my local database somewhere in between. reload_testdata worked fine. I think the issue is that (before this pr) we don't have a cronjob user in our test_data.json (I didn't find it, at least). Thus, when we do reload testdata, we get:

- reset
- migrate (creates cronjob user)
- flush (deletes cronjob user)
- loaddata (doesn't create cronjob user, since it is not in the testdata.json)
- ...

Thus, the migration fails as the user simply does not exist. Can @karyon or @janno42 verify?